src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -357,7 +386,16 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
 .pf-c-toolbar__content-section {
   @include pf-hidden-visible(var(--pf-c-toolbar__content-section--Display));
 
+  align-items: var(--pf-c-toolbar__content-section--AlignItems);
   width: 100%;
+
+  &.pf-m-align-items-center {
+    --pf-c-toolbar__content-section--AlignItems: center;
+  }
+
+  &.pf-m-align-items-baseline {
+    --pf-c-toolbar__content-section--AlignItems: baseline;
+  }
```

Comment:
Same question here as for the group - since it's the default, what's the use case for this class?